PmiRdrModule.php

```diff
@@ -651,13 +657,10 @@ public function rdr_pull($debugApi = false,$singleRecord = false) {
 								$rowData[$redcapField] = $this->getApiValue($dataDetails,$apiField,$metadata[$redcapField]);
 							}
 						}
-	
-						if($testingOnly[$urlKey] == "1") {
-							if(!$debugApi) {
-								echo "<pre>".htmlspecialchars($recordId." => ".var_export($rowData,true))."</pre>";echo "<br />";
-							}
+						if($debugApi) {
+							echo "<pre>".htmlspecialchars($recordId." => ".var_export($rowData,true))."</pre>";echo "<br />";
 						}
-						else {
+						if ($testingOnly[$urlKey] != "1") {
```

Comment:
@kcmcg I am changing this around so we can use `test_rdr`  without having to put the module in test mode, because we shouldn't have to stop the functionality to see a a debug on the test page.